### PR TITLE
Dashboard stats start: days remaining

### LINKED ISSUE #322: [Dashboard Statistic] Days left
Implement the "days left" part of: new mockup

### PR DESCRIPTION
**Pull request by noahsark769**
Fixes #322. Quick change! Starting off the dashboard stats page so that @erictshen and/or @andylouisqin can finish parts of it in their spare time. Other issues for implementing dashboard stats can be found in #108.

screen shot 2015-06-01 at 4 54 46 pm

@sameerav can i get a quick review from you?

### CODE REVIEW

File: revolv/static/dashboard.scss [hunk 1]
```diff
@@ -351,6 +351,21 @@ $sidebar-icon-gray: #999;
...
 
     .project-tabs-content {
+        .days-remaining {
+            // more padding than a normal row
+            padding: 0 40px;
+            text-align: center;
+
+            .num-days-remaining {
+                font-size: 40px;
+                font-weight: bold;
```
Comment (sameerav) [inline revolv/static/dashboard.scss]: just for consistency, should this be a number too? (800)

File: revolv/templates/base/partials/dashboard_project.html [hunk 2]
```diff
@@ -47,4 +45,12 @@ <h2 class="small-10 small-offset-1 columns">{{project.tagline}}</h2>
...
     </div>
+    <div class="content active funding" id="funding">
+        <div class="row donation-stats">
+            <div class="days-remaining small-6 medium-3 large-2 columns">
+                <div class="num-days-remaining">{{project.days_left}}</div>
+                <div class="days-remaining-label">day{{project.days_left|pluralize}} left</div>
+            </div>
+        </div>
+    </div>
 </div>
```
Comment (sameerav) [inline revolv/templates/base/partials/dashboard_project.html]: should there be a newline at the end of files? i don't remember

**Comment by sameerav** [comment]
looks good to me!

File: revolv/templates/base/partials/dashboard_project.html (hunk 2 above)
Comment (noahsark769) [inline revolv/templates/base/partials/dashboard_project.html]: Oh weird pre-commit is supposed to catch that

**Comment by noahsark769** [comment]
Sweet, will fix the font thingy then merge

**Pull request merged**